pkg/config/setup/config.go

```diff
@@ -422,6 +422,7 @@ func InitConfig(config pkgconfigmodel.Config) {
 	config.BindEnvAndSetDefault("network_path.collector.workers", 4)
 	config.BindEnvAndSetDefault("network_path.collector.input_chan_size", 1000)
 	config.BindEnvAndSetDefault("network_path.collector.processing_chan_size", 1000)
+	config.BindEnvAndSetDefault("network_path.collector.pathtest_max_contexts", 10000)
```

Comment:
I'm wondering if `pathtest_contexts_limit` is slightly better since `limit` is commonly used for this kind of upper bound.